_docs/tasks/traffic-management/ingress.md

```diff
@@ -142,7 +142,7 @@ to configure ingress behavior.
 1. Update the secret using `kubectl`
 
    ```bash
-   kubectl delete secret istio-ingress-certs; kubectl create secret tls ingress-secret --key /tmp/tls.key --cert /tmp/tls.crt
+   kubectl create secret tls istio-ingress-certs --key /tmp/tls.key --cert /tmp/tls.crt -n istio-system
```

Comment:
as you'll rev the pr anyway, maybe move the -n closer to kubectl like we do for other examples (a bit odd to have it last arg)

Reply:
moved